test/codeu/chat/server/RawControllerTest.java

```diff
@@ -47,37 +47,25 @@ public void doBefore() {
   @Test
   public void testAddUser() {
 
-    final User user = controller.newUser(userId, "user", Time.now());
-
-    assertFalse(
-        "Check that user has a valid reference",
-        user == null);
-    assertTrue(
-        "Check that the user has the correct id",
-        Uuid.equals(user.id, userId));
+    final User user =
+        controller.newUser(userId, "user", Time.now(), "test passwordHash", "saltCode");
+    assertFalse("Check that user has a valid reference", user == null);
```

Comment:
You can use `assertNotNull`.  A message of just "user" is probably sufficient; it's okay to be a little minimalistic since this message goes to a developer not a user.